_posts/2016-01-23-hair-xray.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: X-rays show hair treatments don't penetrate or fix hair
+author: neuroamanda
+---
+
+_But permanent waving does alter protein structure inside hair_
+
+Miracle products for fussy hair have probably been around longer than hair has even been studied.
+Though hair structure has now been investigated in great detail, with tools like X-ray diffraction and electron microscopy, hair repair products proliferate, and their benefits can be dubious at best.
+Beyond settling this cosmetic question, however, studying hair with imaging methods can be instructive for science &mdash; and health.
+Abnormally curly or twisted hair can actually be indicative of an underlying disease state; [breast cancer](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3086309/), for example, can be accurately predicted using hair samples.
+Disease screening thus might benefit from hair imaging, but it's only useful to study hair if the frequently used hair care products &mdash; shampoo, conditioner, and permanent waving treatments &mdash; don't affect internal hair structure.
+
+In an [earlier study on hair structure](https://doi.org/10.7717/peerj.619), researchers sampled hair from a variety of individuals who used different amounts of shampoo and conditioner.
+In their new study, they wanted to [study just one person's hair](https://doi.org/10.7717/peerj.1296), to discount any effects of differential product use.
+
+Shampoos and conditioners have a whole host of ingredients, many of them completely unnecessary for their primary functions: cleaning and smoothing.
+Both products mainly interact with hair at the surface, or cuticle, level. Shampoo and conditioner will only reach the next layer, the cortex, if the cuticle is very damaged, in which case shampooing may contribute to even _more_ hair damage.
```

Comment:
No strong feelings, but perhaps you could split the sentence at "in which case…" even if you only do so with a semi-colon.